content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -230,30 +226,75 @@ clusters, zones, or regions.
 This model affords the following benefits:
 
 - Improved availability: If a control plane becomes unavailable, the scope of
-  the outage is limited to only that control plane.
+  the outage is limited to only workloads in clusters managed by that control plane.
+
+- Improved configuration isolation: Configuration changes in one control plane affects only
+the workloads in clusters managed by this control plane.
+
+### Control Plane Per Cluster
+
+For maximum availability and configuration isolation you can deploy a control plane in each cluster. Each cluster's control plane
+manages only the workloads in its own cluster. If the control plane is unavailable (due to upgrade or
+temporary outage), the workloads will continue to function with the existing sidecar configuration and
+will re-attempt to connect to the in-cluster control plane periodically.
+
+This model affords the following benefits:
+
+- Highest level of availability: If a control plane becomes unavailable, the scope of
+  the outage is limited to only that cluster and the workloads within it.
+
+- Maximum configuration isolation: You can make configuration changes in one cluster without impacting
+  workloads deployed in any other cluster.
+
+### Availability vs Config Isolation
+
+The following list ranks control plane deployment examples by availability and configuration isolation:
+
+- One control plane per region or a single global control plane (**lowest availability**, **lowest isolation**)
+- Multiple control planes per region
+- One control plane per cluster (**highest availability**, **highest isolation**)
+
+## Mesh models
+
+Istio supports having all of your services in a
+{{< gloss "service mesh" >}}single mesh{{< /gloss >}}, or federating multiple meshes
+together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
+
+### Single mesh
+
```

Comment:
For single mesh, if clusters have different trust domain for each cluster, why aren't those multi-meshes?  Seems it is intuitive to use trust domain to scope the boundary of mesh.

Reply:
Why does trust domain (an opaque string) have to be tied to the CA or mesh? Intuitive is very subjective. To me, it is unintuitive to run single Istio for many clusters or allow pods in one cluster to access internal k8s services of another cluster :). Yet, we have many formulations of this (single/independent/central control planes). The trust domain falls under the same umbrella. You can use a single global trust domain for entire mesh (like central istiod) or many trust domains (one per cluster). Its just essentially build your own adventure, like the rest of this doc.

Reply:
Maybe it doesn't... What I'm trying to understand is how can I differentiate if this is single mesh or multi-meshes, based on your latest doc. From our prior doc, my understanding is that it is based on trust domains and whether admin needs to federate trust bundles.  Now, I don't know, and that is the confusion I had.